python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1226,3 +1226,21 @@ def test_concat_fast():
 def test_func_namespace():
     # note: this test is sensitive to Pandas' internal module layout
     assert xpd.concat is xpd.core.reshape.concat.concat
+
+
+def test_register_accessor():
+    @xpd.api.extensions.register_dataframe_accessor("xyz")
+    class XYZ:
+        def __init__(self, obj):
+            self._obj = obj
+
+        @property
+        def foo(self):
```

Comment:
Would be good to have this test a function that should go through the fast path and a function that should go through like the slow path like

```
def should_be_fast(self):
    return self._obj.dropna()

def should_be_slow(self):
    return self._obj.apply(lambda x: 5)

...

assert df.xyz.should_be_fast ==
assert df.xyz.should_be_slow ==
```